volume/drivers/fake/fake.go

```diff
@@ -275,6 +275,18 @@ func (d *driver) Detach(volumeID string, options map[string]string) error {
 	return nil
 }
 
+func (d *driver) CloudMigrateStart(request *api.CloudMigrateStartRequest) (*api.CloudMigrateStartResponse, error) {
+	return &api.CloudMigrateStartResponse{}, nil
```

Comment:
Resturn the task id that was provided in the request, that way the tests can check if it got to this point.